src/baseService.ts

```diff
@@ -263,17 +263,17 @@ export class BaseService {
   @nonHTTPRequestMethod
   private _resolveQuery(methodName: string, args: any[]): any {
     const meta = this.__meta__;
-    const query = meta[methodName].query || {};
```

Comment:
# Issue 1 - leaking across requests

If method has an `@Queries` decorator, `this.__meta__[methodName].query` exists and is pulled in by reference, so when appending new params to the query object these end up being persisted on the instance and so undesirably get pulled into the next request via `this.__meta__[methodName].query`

`this.__meta__[methodName].query` is only safe for and meant for static query params, cloning the query object before manipulation breaks the reference and fixes this issue